libcxx/include/deque

```diff
@@ -2578,30 +2579,30 @@ inline _LIBCPP_HIDE_FROM_ABI bool operator<=(const deque<_Tp, _Allocator>& __x,
 #  else // _LIBCPP_STD_VER <= 17
 
 template <class _Tp, class _Allocator>
-_LIBCPP_HIDE_FROM_ABI __synth_three_way_result<_Tp>
+_LIBCPP_HIDE_FROM_ABI __synth_three_way_result<_Tp> constexpr
```

Comment:
I think you need to use the macro `CONSTEXPR_SINCE_XXX`.

Reply:
Yeah. This should be replaced with `_LIBCPP_CONSTEXPR_SINCE_CXX26`.